src/targets/gpu/CMakeLists.txt

```diff
@@ -21,8 +21,18 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 # ####################################################################################
-
 list(APPEND CMAKE_PREFIX_PATH /opt/rocm)
+
+if(NOT WIN32)
+    include(MigxLocalTarget)
+    rocm_local_targets_migx(MIGXGPU_TARGETS)
+    if(MIGXGPU_TARGETS)
+        set(GPU_TARGETS ${MIGXGPU_TARGETS} CACHE STRING "GPU targets to compile for")
+        set(AMDGPU_TARGETS ${MIGXGPU_TARGETS} CACHE STRING "AMDGPU targets to compile for")
```

Comment:
Dont set this variable. Its not necessary since you already setting `GPU_TARGETS`.

Reply:
Done.